### ISSUE TITLE
Handling of look-ahead bias

### ISSUE THREAD

**Issue by rhamnett** [jdb78/pytorch-forecasting#51]
Hi, can I please ask how you are handling the potential for look-ahead bias in the scaling of the features etc? This seems to be a common problem in timeseries prediction. I did search the docs but couldn't find any such information. Many thanks.

**Comment by jdb78** [comment]
Hi @rhamnett, 

I should probably make this clearer in the docs and tutorial.

I think there are two answers:

1. Either you do not care too much (maybe because your timeseries is long and stationary) and because you anyways will validate on the validation set which should have no look-ahead bias.
2. Or you use the [`EncoderNormalizer`](https://pytorch-forecasting.readthedocs.io/en/latest/api/pytorch_forecasting.data.encoders.EncoderNormalizer.html#pytorch_forecasting.data.encoders.EncoderNormalizer) that scales dynamically on each encoder sequence as you train.

Generally, I would prefer the `Encoder Normalizer`. However, you might accept look-ahead bias if you are having troubles to find a reasonably stable normalization or you expect a more stable normalization in inference. In the later case, you ensure that you do not learn "weird" jumps that will not be present when running inference, thus training on a more realistic data set. I hope that rational makes sense. Always happy to read a more rigorous analysis. I am sure some statistician has devoted a PhD to the topic.

**Comment by jdb78** [comment]
Added more comments in PR #54
